FinSearchUnified/Tests/Bundle/ProductNumberSearchTest.php

```diff
@@ -367,12 +372,22 @@ public function facetWithVendorFilterProvider()
                     true,
                     'Hersteller',
                     [
-                        new MediaListItem('Anderson, Gusikowski and Barton', 'Anderson, Gusikowski and Barton', true),
-                        new MediaListItem('Bednar Ltd', 'Bednar Ltd', false)
+                        new MediaListItem(
+                            'Anderson, Gusikowski and Barton',
```

Comment:
Is the name also the id?

Reply:
Yes Shopware works with ids. The only difference would be special characters, which we have tests for. The ID can only contain data that can be written in an HTML name attribute.

Reply:
This is from the HTML source:

![image](https://user-images.githubusercontent.com/20555112/104601912-1ba9a180-567b-11eb-92fb-e06181130cd1.png)

The id represents the name as escaped data and the name represents the actual shown value. If I recall correctly in Shopware 5, this will be done automatically by Shopware.